rpm/mapplauncherd-qt5.spec

```diff
@@ -4,9 +4,9 @@ Summary:    Application launch boosters for Qt5
 Version:    1.1.18
 Release:    1
 License:    LGPLv2+
-URL:        https://github.com/nemomobile/mapplauncherd-qt/
+URL:        https://github.com/sailfishos/mapplauncherd-qt/
 Source0:    %{name}-%{version}.tar.bz2
-Requires:   mapplauncherd >= 4.1.0
+Requires:   mapplauncherd >= 4.2.3
```

Comment:
Upping the main dependency along with devel dependency would not tickle any whiskers, but dropping devel package version requirement when dealing with something that is effectively a build time thing makes me slightly confused...

Maybe depending on main package somehow produces easier to grok error messages during transition from "devel packages do not have pc files" to "have pc files" when pkgconfig build deps are used. But that kind of requires making assumptions about deps between mapplauncherd and mapplauncherd-devel, or?

So, IMO having a minimum devel version requirement - "BuildRequires:  pkgconfig(applauncherd) >= 4.2.3"  - or so might make things more clear and explicit.

NB The other PRs seem to drop version requirements both for devel and main packages.

Reply:
Added versioned dependencies for mapplauncherd.